test/e2e/tests/confirmations/transactions/shared.ts

```diff
@@ -58,6 +58,34 @@ export async function confirmContractDeploymentTransaction(driver: Driver) {
   );
 }
 
+export async function confirmRedesignedContractDeploymentTransaction(
```

Comment:
This is a nice abstraction, but I'm really excited for full adoption of the page object model so we can vastly simplify all of these tests and avoid tons of duplication across different helpers and tests etc.

Reply:
Agreed!